### PR TITLE
First PR from this branch

### PR DESCRIPTION
**Pull request by yashanktiwari**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,15 @@
-Test document
\ No newline at end of file
+Test document
+
+adding some new lines
+
+to test the functionality
+
+of the comments that can be written
+
```
Comment (yashankt-philips) [inline test.txt:8]: Why you have done this?

File: test.txt [hunk 2]
```diff
@@ -1 +1,15 @@
...
+on the pull request.
+
+And then try to resolve them later
+
+by writting and then again updating the code
+
+in the PR.
```
Comment (yashankt-philips) [inline test.txt:15]: Change this also

**Review by yashankt-philips — CHANGES_REQUESTED** [review]

File: test.txt (hunk 2 above)
Comment (yashanktiwari) [reply test.txt:15]: It is perfect only.

File: test.txt (hunk 1 above)
Comment (yashanktiwari) [reply test.txt:8]: Made the appropriate changes

**Pull request merged**